supported-currencies.json

```diff
@@ -0,0 +1,544 @@
+{
+    "supported-currencies": [
+      {
+        "Currency Name": "United Arab Emirates Dirham",
+        "ISO Code": "AED",
+        "Exponent": "2"
+      },
+      {
+        "Currency Name": "Albanian lek",
+        "ISO Code": "ALL",
+        "Exponent": "2"
+      },
+      {
+        "Currency Name": "Armenian dram",
+        "ISO Code": "AMD",
+        "Exponent": "2"
```

Comment:
use small case for keys

Reply:
fixed here https://github.com/razorpay/razorpay-quick-payments/blob/b7757b7948021cd22c1eb9d5326e5ea8c3decb1b/supported-currencies.json#L6